{
  "contest": 261,
  "handle": "wisdomn_",
  "risk": "3",
  "title": "Functionn withdrawToChain is vulnerable to Unauthorized calls Attack.",
  "issueId": 458,
  "issueUrl": "https://github.com/code-423n4/2023-07-tapioca-findings/issues/458"
}